DealOrNoDeal/src/exercise/Case.java

```diff
@@ -0,0 +1,29 @@
+package exercise;
+
+public class Case {
+    private int number;
+    private int value;
+
+    public Case(int number, int value){
+        this.number = number;
+        this.value = value;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + (number+1) +
+                "]";
+    }
+
+    public int getNumber() {
+        return number;
+    }
+
+    public void setNumber(int number) {
+        this.number = number;
+    }
+
+    public int getValue(){
+        return value;
+    }

```

Comment:
Well done! your case is well encapsulated as an object and your methods are well organized by the different tasks 
Only the error handling regarding  "Do you accept the offer?" could have been more accurate